src/builder/Actions.sol

```diff
@@ -593,7 +593,8 @@ library Actions {
 
         // Construct QuarkOperation
         IQuarkWallet.QuarkOperation memory quarkOperation = IQuarkWallet.QuarkOperation({
-            nonce: accountState.quarkNextNonce,
+            nonce: bytes32(uint256(accountState.quarkNextNonce)),
```

Comment:
There's a bunch of interface changes that need to be made in `QuarkBuilder` for the client to pass in a random nonce. I plan to do that as a separate PR.